package.json

```diff
@@ -55,6 +56,7 @@
     "electron-log": "4.4.8",
     "electron-window-state": "5.0.3",
     "fs-extra": "11.1.0",
+    "lodash": "^4.17.21",
```

Comment:
Oh wow, I thought we already had `lodash` in the project. That's why I suggested it. Thanks for adding it.